.github/workflows/e2e_cuda.yml

```diff
@@ -4,21 +4,18 @@ on:
   schedule:
```

Comment:
should we still care about schedule?

Reply:
No, we should just get rid of the nightly.

Reply:
Done.